src/test/java/systemtests/AddressBookSystemTest.java

```diff
@@ -39,8 +33,8 @@
 import seedu.address.model.AddressBook;
 import seedu.address.model.Model;
 import seedu.address.testutil.TypicalPersons;
-import seedu.address.ui.BrowserPanel;
 import seedu.address.ui.CommandBox;
+import seedu.address.ui.InfoPanel;
 
 /**
  * A system test class for AddressBook, which provides access to handles of GUI components and helper methods
```

Comment:
Can change AddressBook to HR+ :)

Reply:
We can have a separate PR to convert all AddressBook to HR+ 😄

Reply:
👌🏼👌🏼